cmake/tcmalloc.h.in

```diff
@@ -76,6 +74,12 @@
 # endif
 #endif
 
+#if defined(__cpp_aligned_new) || \
```

Comment:
question: just cpp_aligned_new not enough?

Reply:
So there's some uncertainty here on MSVC's behaviour.

As you may know, MSVC will set `__cplusplus` to the wrong value unless [`/Zc:__cplusplus`](https://learn.microsoft.com/en-us/cpp/build/reference/zc-cplusplus) is also provided.
This is unfortunately still the default behaviour on the command line.

Now, I can't find an authoritative answer whether the C++ feature-test macros are also affected by this unfortunate situation.

That's why we have that additional `_MSVC_LANG` check, which serves the same role as `__cplusplus` - but never lies.